### ISSUE TITLE
Feature: Keep file size / line numbers.

### ISSUE THREAD

**Issue by ibc** [FormidableLabs/gulp-html-extract#9]
When piping the extracted content to `jshint`, `eslint` or whatever, the reporter references line numbers that, obviously, do not match the line in the original (physical) source file.

Would it make sense that, rather than just piping the extracted content, this library would keep empty lines for the non-extracted content so file numbers match?

**Comment by ryan-roemer** [comment]
@ibc -- That's a fantastic idea and I'd love a PR if you have time to work one up. Otherwise, I'll take a stab at it in a while...

As a strawman, I'm thinking something like a new `opts.pad` like:

``` js
/**
 * ## `html-extract`
 *
 * Extract text from HTML.
 *
 * @param {Object} opts       Options
 * @param {String} opts.sel   Element selector [Default: `script`]
 * @param {String} opts.strip Strip to indent level [Default: `false`]
 * @param {String} opts.pad   Pad text to start line in source file [Default: `false`]
 * @returns {void}
 */
```

I'm not sure if cheerio already captures line numbers / starting line information for code chunks, but if so, this would be a super-easy feature to add!

**Comment by ibc** [comment]
If you call `cheerio.load(file.contents.toString("utf8"), { withStartIndices: true });` every found element will have a `startIndex` property pointing to the exact char position of the beginning of the tag.

**Comment by ibc** [comment]
Working on a PR.

**Comment by ibc** [comment]
Done.

**Comment by ryan-roemer** [comment]
Implemented in: https://github.com/FormidableLabs/gulp-html-extract/pull/10

Released in `gulp-html-extract@0.2.0`

**Comment by ibc** [comment]
Great, thanks a lot!
